client/src/app/components/title-bar/nav-menu/nav-menu.component.css

```diff
@@ -1,3 +1,7 @@
 .menu { 
     cursor: pointer;
+}
+
+a {
+    text-decoration: none;
```

Comment:
Yo, this probably needs to be up higher, like in the main stylesheet for the entire app.  I think we want all links to be like this.

Reply:
Excellent idea!

Reply:
I see that you removed it from this file, but it doesn't look like the styles.css file was pushed up?